ci-operator/config/openshift/release/openshift-release-master__ci-4.8-upgrade-from-stable-4.7.yaml

```diff
@@ -114,6 +114,11 @@ tests:
     - ref: ipi-conf-user-workload-monitoring
     - chain: ipi-aws-pre-stableinitial
     workflow: openshift-upgrade-aws
+- as: e2e-aws-upgrade-infra
+  interval: 72h
+  steps:
+    cluster_profile: aws
+    workflow: openshift-upgrade-aws-infra
```

Comment:
[Is][1]:

```
Monitoring pods didn't move to infra nodes
```

something we expect to fix on 4.7?  If so, is there a bug tracking it?  If not, can we drop this job that will always fail?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/21175/rehearse-21175-periodic-ci-openshift-release-master-ci-4.8-upgrade-from-stable-4.7-e2e-aws-upgrade-infra/1465668714648047616#1:build-log.txt%3A117